packages/nx/src/command-line/show/projects.ts

```diff
@@ -81,8 +84,6 @@ export async function showProjectsHandler(
       console.log(project);
     }
   }
-
-  await output.drain();
```

Comment:
This has the potential to miss some output, we added it to fix an issue. What is the reason for this being reverted?